src/lem/store.rs

```diff
@@ -395,6 +412,27 @@ impl<F: LurkField> Store<F> {
         Some([sym, val, env])
     }
 
+    #[inline]
+    pub fn push_provenance(&self, sym: Ptr, val: Ptr, env: Ptr) -> Ptr {
+        assert_eq!(*sym.tag(), Tag::Expr(Cons));
+        //assert_eq!(*env.tag(), Tag::Expr(Env));
```

Comment:
Is this left as a suggestive TODO? In that case, it's better to explain what we're waiting for